packages/react-ssr/package.json

```diff
@@ -43,7 +43,7 @@
     "gcds.css"
   ],
   "scripts": {
-    "prebuild": "del-cli --force ./dist && copyfiles -f ../web/dist/components/*.js ./dist/esm/components && make-dir ./dist/esm && make-dir ./dist/types",
+    "prebuild": "del-cli --force ./dist && copyfiles -f ../web/dist/components/*.js ./dist/esm/components && copyfiles -f ../../node_modules/@stencil/core/internal/client/index.js ./dist/esm/lib/stencil && copyfiles -f ../../node_modules/@stencil/core/internal/client/shadow-css.js ./dist/esm/lib/stencil && make-dir ./dist/esm && make-dir ./dist/types",
```

Comment:
Let's add this to the README.md to give them a heads up about this prebuild step.

Reply:
This is only happens on our end when we build the package, do we need to include that in the readme?

Reply:
You're right ~ realizing that now. We don't need to mention the specific details like the prebuild step but just a brief mention that the package uses a patched version of stenciljs would be helpful for transparency.